packages/bridge/src/nitro.ts

```diff
@@ -232,7 +232,8 @@ export async function setupNitroBridge () {
       // We use only object form of alias in base config
       if (Array.isArray(config.resolve.alias)) { return }
       config.resolve.alias.ufo = 'ufo/dist/index.mjs'
-      config.resolve.alias.ohmyfetch = 'ohmyfetch/dist/index.mjs'
```

Comment:
It wouldn't be a bad idea to keep this for any packages still using old name. I also plan to create a placeholder stub releases for `ohmyfetch`

Reply:
oops, that was what the second line was meant to be.